lib/data-types.js

```diff
@@ -603,6 +603,16 @@ DATEONLY.prototype._isChanged = function _isChanged(value, originalValue) {
   return true;
 };
 
+function INTERVAL() {
+  if (!(this instanceof INTERVAL)) return new INTERVAL();
+}
+util.inherits(INTERVAL, ABSTRACT);
+
+INTERVAL.prototype.key = INTERVAL.key = 'INTERVAL';
+INTERVAL.prototype.validate = function validate() {
+  return true;
```

Comment:
I've not found a good way to validate intervals. 

See https://stackoverflow.com/questions/38692972/how-to-validate-a-moment-js-duration for an example of this not working.

Reply:
Actually, is `validate` called by anything? I was halfway through updating this to validate ISO and postgres interval strings when I realized nothing seems to call these functions. 

Is that the case?